lib/assessment.md

```diff
@@ -83,7 +111,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+color: yellow;
+font-family: sans-serif;
```

Comment:
Is it because of specificity? But I though that CSS rules that come 'later' (lower in the file) beat earlier ones.